gradle.properties

```diff
@@ -1 +1 @@
-version = 1.7.0-SNAPSHOT
+version = 2.0.0-beta1-SNAPSHOT
```

Comment:
beta1 is OK with semantic versioning. Spring uses rc1, rc2. Hibernate uses Beta1, Beta2.